### PR TITLE
add perf cli smoke test

### PR DESCRIPTION
**Pull request by ayrat555**
This PR adds ci step that runs `transactions` perf test

### CODE REVIEW

**Comment by ayrat555** [comment]
ci fails because `feefeed` is broken

File: .circleci/config.yml [hunk 1]
```diff
@@ -880,6 +880,31 @@ jobs:
       - run: make docker-perf IMAGE_NAME=$PERF_IMAGE_NAME
       - run: IMAGE_NAME=$PERF_IMAGE_NAME sh .circleci/ci_publish.sh
 
+  run_perf_cli_smoke_test:
+    machine:
+      image: ubuntu-1604:201903-01
+    environment:
+      PERF_IMAGE_NAME: "omisego/perf:ci"
```
Comment (boolafish) [inline .circleci/config.yml:887]: where is generating the `:ci` tagged image?? Also when will this tag get updated?

Comment (ayrat555) [reply .circleci/config.yml:887]: it's a name for the temporary image used only in circle ci.
Generation is here - https://github.com/omgnetwork/elixir-omg/pull/1762/files/4ee5857447db0e44a4b6c70663d41ab015d0d902#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R901

Comment (boolafish) [reply .circleci/config.yml:887]: ah my bad....it is just local image where I somehow thought it was from remote.

**Review by boolafish — APPROVED** [review]
The step itself looks good to me. Do you think there is potential to just merge `test_docker_compose_performance` with this (?) They seems overlapping a bit on the purpose of it.

File: .circleci/config.yml (hunk 1 above)
Comment (InoMurko) [reply .circleci/config.yml:887]: what is true is the convention is to name the `current` image - `latest` :)

**Comment by InoMurko** [comment]
> The step itself looks good to me. Do you think there is potential to just merge `test_docker_compose_performance` with this (?) They seems overlapping a bit on the purpose of it.

I would second that!

**Review by InoMurko — APPROVED** [review]

**Pull request merged**